web_searchbar_full_width/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Full width searchbar",
+    "version": "12.0.1.0.0",
+    "author": "Therp BV, "
+              "Odoo Community Association (OCA), "
+              "Siddharth Bhalgami",
+    "license": "AGPL-3",
+    "category": "Tools",
```

Comment:
category = web

Reply:
IMO category will remain tools as this module is not adding any new functionality to the web